src/app/collections/collection-sheet-status/collection-sheet-status.component.html

```
@@ -0,0 +1,251 @@
+      </mat-card>
+    </div>
+
+    <!-- Retry button for failed -->
+    <div *ngIf="summary.failed > 0" style="margin-bottom: 16px">
+      <button mat-raised-button color="warn" (click)="retryAllFailed()" [disabled]="isRetrying">
+        <mat-icon>refresh</mat-icon> Retry All Failed
+      </button>
+    </div>
+
+    <!-- Group-wise Breakdown -->
+    <div *ngFor="let group of groups" style="margin-bottom: 24px">
```

HTMLHint (1.9.1)

[warning] Inline style [  style="margin-bottom: 24px" ] cannot be used.

(inline-style-disabled)